actionpack/lib/action_dispatch/middleware/host_authorization.rb

```diff
@@ -83,7 +119,7 @@ def initialize(app, hosts, deprecated_response_app = nil, exclude: nil, response
         response_app ||= deprecated_response_app
       end
 
-      @response_app = response_app || DEFAULT_RESPONSE_APP
+      @response_app = response_app || DefaultResponseApp.new(consider_all_requests_local)
```

Comment:
I don’t think you need to pass consider_all_requests_local. It should be available in the env/request. 
See: https://github.com/rails/rails/pull/42828#issuecomment-884470990

Reply:
fixed